### ISSUE TITLE
Sats for a transcript

### ISSUE THREAD

**Issue by kouloumos** [bitcointranscripts/bitcointranscripts#171]
I see bitcointranscripts as a highly beneficial project for the Bitcoin community so I am always trying to think of ways to increase awareness and possibly attract more contributors. I am opening this to discuss increasing awareness by applying for an [education grant](https://geyser.fund/project/bitcoin-education) through the latest [Geyser Grants](https://geyser.fund/grants) round.

An idea that I had for a bit is bounties for transcripts. An incentive for people to get their first sats while [learning about Bitcoin through transcription](https://bitcoinmagazine.com/culture/op-ed-want-to-learn-about-bitcoin-try-contributing-a-transcript). My idea was that those bounties could be funded by entities that would put a bounty for their own content or even crowdfunded by the community in case of a valuable technical/dev transcript. Before Geyser announced this latest grants initiative, I was actually thinking that this crowdfunding part could be achieved [using their existing platform](https://geyser.fund/launch) to create small crowdfunding campaigns for individual transcripts. This is still on the table, but the focus of this is their grants initiative. 

My idea is that we can apply for a grant that would allow us to give a predefined amount of Bitcoin to new contributors after their first transcript. It will possibly not be enough for their labour, but that's not the point. The goal is to give an extra incentive for people to get involved into an open-source project, provide value to the community and possibly get their first sats while learning about Bitcoin. 

Any thoughts?

You can see the existing grant applications [here](https://airtable.com/shrfeI21FWzyCqHZy/tblwlFBSxMvV0JhzU).

**Comment by adamjonas** [comment]
I'm not opposed to the experiment but I also don't think this is dependent on geyser or https://bitcoinbounties.org/, opensats or whatever else. I don't really like the bounty model for tasks as it becomes subjective when they are up to the expected standard and then people get upset. On the other hand, I really like the idea of encouraging people to transcribe talks since it's a great entry point and adds value immediately. I'd be fine playing around with some ideas if you are motivated to drive it.

**Comment by kouloumos** [comment]
Yes, the main idea behind this "sats for a transcript" idea is encouraging people to transcribe. Giving that extra incentive is how I though the encouragement could have more success while bringing more awareness to the project. 
I see this "reward" idea as direct encouragement compared to passively encouraging people through awareness initiatives, like the https://github.com/bitcointranscripts/bitcointranscripts/pull/168 tweet-bot experiment.

I am happy to discuss more ideas for direct and indirect encouragement and I can definitely drive it. This project provides value as it is, but imo it still has room to grow, either with [website improvements](https://github.com/bitcointranscripts/bitcointranscripts.github.io/pull/42), or with this kind of awareness experiments.

**Comment by kouloumos** [comment]
**Update:**
[I ended up applying for this grant for Bitcoin Transcripts](https://airtable.com/shrLwYSAvD5pO9gjG/tblwlFBSxMvV0JhzU/viweup7YUXcH88QwK/recBnJvr1mAzF0ddh) and[ it seems that we made it into the selected projects](https://twitter.com/geyserfund/status/1567538531198799873). It's not much but we'll have 2.5M sats to run a few awareness experiments in the future. They haven't communicate on how they will distribute the funds, I'll give an update to discuss custody as soon as they reach out.

**Comment by kouloumos** [comment]
Email Update from Geyser:

> Hey Grant applicant,
> 
> Congratulations! You have been awarded a Geyser Grant for our very first Grant giveaway.
> 
> Grant Group 2: 2.5M Satoshis
> 
> Next Steps:
> 1. Share your Bitcoin on-chain public address with us so we can send you the funds. 
> We will only accept an address sent from your very own email. This does not apply to projects that are live on Geyser - we can send you the funds directly through your Geyser campaign.
> 2. Check & Retweet our announcement on [Twitter](https://twitter.com/geyserfund/status/1567537222005530625?s=20&t=QB5JrZdp2obGDFGXH0gY5Q).
> 3. Join our [Twitter Space](https://twitter.com/i/spaces/1jMJgLgkRpwxL?s=20) tomorrow at 12PM EST 
> 
> It will take us roughly 1-2 weeks to process the payment.
> 
> Cheers,
> Mick

I would prefer this to be a 2/3 multisig with @adamjonas and @michaelfolkson. If you believe that this adds a lot of burden, I could deal with the funds (at least for now) but that's not my ideal scenario.

**Comment by michaelfolkson** [comment]
@kouloumos: 2.5 million satoshis is 0.025 Bitcoin so ~500 USD at today's price. I don't know what Jonas thinks but I think a multisig setup is overkill for that amount, I'm happy to trust you to store it and distribute it as you see fit. Unless you want to experiment with multisig for educational/experimental purposes :)

**Comment by adamjonas** [comment]
@kouloumos I don't have any interest in being involved in the custody of these funds.

**Comment by kouloumos** [comment]
I appreciate your responses. Yes, probably an overkill at this stage. I've asked for the funds to be send at [bc1qnznqkm3xdz4mjfs4f7x0ahmen3tdnsyqjuevzw](https://mempool.space/address/bc1qnznqkm3xdz4mjfs4f7x0ahmen3tdnsyqjuevzw).

The overview of my idea on how those funds could be utilized is on the original post. I'm not planning to take any actions soon due to other priorities, I'll reply for anything related.

Those are roughly the steps that I have in mind for this initiative:

1. Extract metadata for all transcripts to utilize https://github.com/bitcointranscripts/bitcointranscripts.github.io/pull/42 together with minor UI changes as mentioned [here](https://github.com/bitcointranscripts/bitcointranscripts/pull/180#discussion_r929028332) to utilize new metadata and improve discoverability [WIP]. 
3. Create an umbrella issue for suggestions on potential transcripts. Next, transcription software (e.g [otter](https://otter.ai/)) can be used to create a first draft of the transcript for which a "good-first-transcript" issue (such as https://github.com/bitcointranscripts/bitcointranscripts/issues/82) could be created. I believe that this can help new contributors to start (that's how I did my first transcripts). 
4. Introduce a CTA for the "sats for a transcript" initiative at the homepage.

**Comment by MickMorucci** [comment]
The Transcription bounties idea is an interesting one. 
Grants are more retro-active - after the event.
Bounties are pro-active - before the event.


My only concern is the low use of  https://bitcoinbounties.org/, last bounty goes back to Feb 2022!
That said it makes a lot of sense for us to integrate that in based on our existing ecosystem of projects, etc. We'll be looking into adding this in the future. If anyone is interested in helping us build this feature, please do reach out! (@geyserfund on twitter)

@kouloumos , would also love to chat as I think your project related to transcription service would be very related to what we're building next. Would love to have a chat, here is my Calendly: https://calendly.com/michele-morucci/30min

**Comment by adamjonas** [comment]
@kouloumos is this safe to close? I think we have a real plan now.

**Comment by kouloumos** [comment]
Closing, for reference see https://review.btctranscripts.com/. 
For transparency: [part of the funds](https://mempool.space/tx/babe23ab5d2664f54d1ae3a39c968f23b8924f48e3c15709ac0a69decf0ff22c) received from Geyser Grants is being used for the transcription-review project, with the rest of it potentially going to the same initiative.
